acceleration_handler_node:
  ros__parameters:
    lap_count_topic: /lap_count
    velocity_estimation_topic: /vcu_msgs/velocity_estimation
    mission_finished_topic: /vcu_msgs/mission_finished
    brake_controller_topic: /control/brake_controller
    check_distance_frequency: 100.0
    mission_finished_velocity_threshold: 1.0
    lateral_deviation_threshold_emergency: 1.0 #1.5
    yaw_threshold_emergency: 0.5
